packages/backend/deployments/local/Containerfile.development

```diff
@@ -5,6 +5,7 @@ USER root
 
 # Install development tools
 RUN dnf install -y git postgresql && \
+    curl -sSf https://atlasgo.sh | sh && \
```

Comment:
that's risky toinstall from remote url and will not working Konflux hermetic builds. Can't we have this packaged ? Or part of codebase ?